pkg/engine/systemd.go

```diff
@@ -8,9 +8,9 @@ import (
 	"k8s.io/klog/v2"
 )
 
-func systemdPodman(ctx context.Context, path, dest string, target *api.Target) error {
+func systemdPodman(ctx context.Context, path, dest string, target *api.Target, prev *string) error {
```

Comment:
is a ptr to a string passed because the string value can be very large, or because its value is modified by the receiver?

Reply:
prev is meant to be an optional argument to the function, is there a better way to do optional args in go other than using pointers?

Reply:
That's a good use of a ptr arg. "" is sometimes used for the same purpose but that can't be distinguished from a real empty value, so a ptr is better. _Sometimes_ (especially if you think there could be additional string args added) you could make this last arg variadic and then callers don't have to pass it when it's nil.